### PR TITLE
Feauture

### PR DESCRIPTION
**Pull request by ijosue16**
#  Hello microverse project 
added HTML and CSS file.
added linters.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```
Comment (levy002) [inline index.md]: - [ ] KIndly make sure the `test.md` file is ignored by git and is not present in your GitHub repository in order to meet all project's requirements :wink: 
- [ ] Kindly make sure to rename it `test.md` in order to meet the requirements :wink:

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @ijosue16   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to add a descriptive PR title and PR description. It makes your repo looks more professional. :wink: 

   - Example of good PR
image

- [ ] KIndly make sure to clone the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) in your repo in order to make the repo looks more professional and try to customize it according to your project. 

   **_Make sure your README looks more professional by defining the headers of each section. Kindly follow the provided template to achieve it_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly make sure to write your meaningful commit message in imperative tense and capitalized it. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.md (hunk 1 above)
Comment (ijosue16) [reply index.md]: @levy002 I have made changes

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @ijosue16  ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions

- [ ] Kindly add a Readme file to your project. You may use this [Readme template](https://github.com/microverseinc/readme-template) to guide you. Kindly use [markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) to achieve this.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ijosue16** [comment]
merging the feature branch to the main branch

**Pull request merged**